packages/collector/test/tracing/database/couchbase/app.mjs

```diff
@@ -10,7 +10,7 @@ import express from 'express';
 import { v1 } from 'uuid';
 import morgan from 'morgan';
 import requestPromise from 'request-promise';
-import request from 'request-promise-native';
+const fetch = require('node-fetch');
```

Comment:
issue: This needs to be an import statement.

Reply:
resolved